nucleus/security/core/src/main/java/com/sun/enterprise/security/auth/login/LDAPLoginModule.java

```diff
@@ -73,7 +73,6 @@
  *   are taken as group names in which the user has membership. If no entries are found, the group membership is empty.
  * </ol>
  *
- *
```

Comment:
Is this the change that requires a copyright header? If so I wouldn't change the file at all. This comment applies to quite a number of the touched files.

Reply:
Okay, fair point